x-pack/plugins/ml/common/constants/validation.ts

```diff
@@ -17,3 +17,10 @@ export const SKIP_BUCKET_SPAN_ESTIMATION = true;
 export const ALLOWED_DATA_UNITS = ['B', 'KB', 'MB', 'GB', 'TB', 'PB'];
 
 export const JOB_ID_MAX_LENGTH = 64;
+
+// Data Frame Analytics
+export const TRAINING_DOCS_UPPER = 200000;
+export const TRAINING_DOCS_LOWER = 200;
+export const INCLUDED_FIELDS_THRESHOLD = 100;
+export const MINIMUM_NUM_FIELD_FOR_CHECK = 25;
+export const PERCENT_EMPTY_LIMIT = 0.3;
```

Comment:
`FRACTION_EMPTY_LIMIT` might be a better name here, as it is really checking for `30%` not `0.3%`

Reply:
Good point - updated in 64b4e9dd5da186820cd28990b244d906bc3d3964